tests/esm-path-mapping/node_modules/ambient/index.js

```diff
@@ -0,0 +1 @@
+export default 'ambient';
```

Comment:
Node only accepts ESM export syntax if module is in ESM mode; see next comment.

Reply:
Good catch!